tests/components/ios/test_init.py

```diff
@@ -26,20 +26,15 @@ def mock_dependencies(hass):
 
 async def test_creating_entry_sets_up_sensor(hass):
     """Test setting up iOS loads the sensor component."""
+    # so it will not have an empty config
+    await bootstrap.async_from_config_dict({"foo": "bar"}, hass)
+    entry = MockConfigEntry(domain=ios.DOMAIN, data={},)
+    entry.add_to_hass(hass)
     with patch(
         "homeassistant.components.ios.sensor.async_setup_entry",
         return_value=mock_coro(True),
     ) as mock_setup:
-        result = await hass.config_entries.flow.async_init(
-            ios.DOMAIN, context={"source": config_entries.SOURCE_USER}
-        )
-
-        # Confirmation form
-        assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
-
-        result = await hass.config_entries.flow.async_configure(result["flow_id"], {})
-        assert result["type"] == data_entry_flow.RESULT_TYPE_CREATE_ENTRY
-
+        assert await hass.config_entries.async_setup(entry.entry_id)
```

Comment:
I think this should work.
```suggestion
        assert await async_setup_component(hass, ios.DOMAIN, {ios.DOMAIN: {}})
```

Reply:
not sure i fully understood this test case, but i thought that the point is that it gets loaded from a config entry vs. async_set_component that is used in the other tests, and in this case, i assume config will not have anything for the key ios.DOMAIN.
what am i missing here? :)

Reply:
The config entry will be loaded by `async_setup_component` too.